src/pages/en/guides/migrate-to-astro/from-nuxtjs.mdx

```diff
@@ -2,68 +2,682 @@
+
+### Migrating Tests
+
+As Astro outputs raw HTML, it is possible to write end-to-end tests using the output of the build step. Any end-to-end tests written previously might work out-of-the-box, if you have been able to match the markup of your Nuxt site. Testing libraries such as Jest and Vue Testing Library can be imported and used in Astro to test your Vue components.
+
+See Astro's [testing guide](/en/guides/testing/) for more.
+
+## Reference: Convert NuxtJS Syntax to Astro
+
+### Nuxt Variable Interpolation to Astro
+
+A Nuxt Vue component might have the following to display a variable within a paragraph tag:
+
+```vue title="components/Component.vue"
+<p>{{message}}</p>
+```
+
+To interpolate a variable in an Astro component, change the set of two curly braces to one set of curly braces:
+
+```astro title="src/components/Component.astro"
+<p>{message}</p>
+```
+
+### Nuxt Property Passing to Astro
+
+
+To bind a property to an attribute or component in Vue, you'd do the following:
+
+```vue title="components/Component.vue"
+<p v-bind:aria-label="message">...</p>
+<!-- Or -->
+<p :aria-label="message">...</p>
+<!-- Also support component props -->
+<Header title="Page"/>
+```
+
+To bind an attribute or component property in an Astro component, change this syntax to the following:
+
+```astro title="src/components/Component.astro"
+<p aria-label={message}>...</p>
+<!-- Also support component props -->
+<Header title={"Page"}/>
+```
+
+### Nuxt Links to Astro
+
+Convert any Nuxt `<NuxtLink to="">` components to HTML `<a href="">` tags. 
+
+```astro del={1} ins={2}
+<NuxtLink to="/blog">Blog</Link>
+<a href="/blog">Blog</a>
+```
+
+Astro does not use any special component for links, although you are welcome to build custom link components. You can then import and use this `<Link>` just as you would any other component.
+
+```astro title="src/components/Link.astro"
+---
+const { to } = Astro.props
+---
+<a href={to}><slot /></a>
+```
+
+### Nuxt Imports to Astro
+
+If necessary, update any [file imports](/en/guides/imports/) to reference relative file paths exactly. This can be done using [import aliases](/en/guides/typescript/#import-aliases), or by writing out a relative path in full. 
+
+Note that `.astro` and several other file types must be imported with their full file extension.
+
+```astro title="src/pages/authors/Fred.astro" ".astro"
+---
+import Card from `../../components/Card.astro`;
+---
+<Card />
+```
+
+### Nuxt Dynamic Page Generation to Astro
+
+In Nuxt, to generate a dynamic page you either must:
 
-- Add your own!
\ No newline at end of file
+- Use SSR
+- [Use the `generate` function in `nuxt.config.js` to define all possible static routes](https://nuxtjs.org/docs/configuration-glossary/configuration-generate/)
+
+In Astro, you must export a `getStaticPaths` function to tell the framework which routes it is expected to generate for a dynamic page.
```

Comment:
Digging this, but makes no mention of SSR in Astro. So, I think we need to find a way to also include it.

Reply:
I was originally thinking something like:

```suggestion
In Astro, you must export a `getStaticPaths` function to tell the framework which routes it is expected to generate for a dynamic page. This not only helps static site generation but also allows Astro to speed up its own server-side rendering.
```

With a link to the SSR docs page, but I'm not sure of its technical accuracy. What do y'all think @sarah11918 @Jutanium ?